src/js/media-chrome-listitem.js

```diff
@@ -34,6 +39,14 @@ class MediaChromeListitem extends window.HTMLElement {
     shadow.appendChild(listitemHTML);
   }
 
+  set value(value) {
```

Comment:
This could be useful for either case. For example, you could have an image for each of the list items, but you'd want the value to be a number or something. This matches `<option>`'s `value` attr and propr.

Reply:
I'm sold.